CMakeLists.txt

```diff
@@ -72,6 +72,16 @@ endif()
 # CPACK_DEBIAN_PACKAGE_VERSION was previously a cache variable. Evict it.
 unset(CPACK_DEBIAN_PACKAGE_VERSION CACHE)
 
+if (NOT CPACK_PACKAGING_INSTALL_PREFIX)
+  set(CPACK_PACKAGING_INSTALL_PREFIX "/" CACHE STRING "Package root path. \"/\" is correct for normal package builds.." FORCE)
+  message(STATUS "Setting unspecified CPACK_PACKAGING_INSTALL_PREFIX to '${CPACK_PACKAGING_INSTALL_PREFIX}'. This is the correct setting for normal builds.")
+endif()
+
+list(APPEND CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "${CPACK_PACKAGING_INSTALL_PREFIX}var/lib/irods")
+list(APPEND CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "${CPACK_PACKAGING_INSTALL_PREFIX}var/lib/irods/scripts")
+list(APPEND CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "${CPACK_PACKAGING_INSTALL_PREFIX}var/lib/irods/scripts/irods")
+list(APPEND CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "${CPACK_PACKAGING_INSTALL_PREFIX}var/lib/irods/scripts/irods/test")
```

Comment:
Don't forget to use the recommendation provided by @SwooshyCueb as seen here.
https://github.com/irods/irods_rule_engine_plugin_python/blob/8f7356e499e60a4e55475506b0078a02d115f842/CMakeLists.txt#L126-L131

Notice, the use of `include(IrodsCPackCommon)` as well as the `list(APPEND ...)` lines.

Reply:
the tiering CMakeLists.txt already has such an include, but in a different order. I may be missing the sublety here. @SwooshyCueb

Reply:
`var/lib/irods` -> `${IRODS_HOME_DIRECTORY}` in all cases

Reply:
@alanking  - Does that mean it should read as follows?:
```list(APPEND CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "${CPACK_PACKAGING_INSTALL_PREFIX}${IRODS_HOME_DIRECTORY}")
```

Reply:
Done .